code/scripts/cards.js

```diff
@@ -0,0 +1,21 @@
+// Function to dynamically return a card template based on character object data
+export const getCard = (person) => {
+  let html = `
```

Comment:
You are creating it once, and then you do not change it so it should be created as a `const`